### ISSUE TITLE
[javascript] Field declaration should end with ';'

### ISSUE THREAD

**Issue by ris58h** [antlr/grammars-v4#3546]
Code:
```
class Foo {
    bar baz;
}
```

The code isn't correct according to documentation https://tc39.es/ecma262/multipage/ecmascript-language-functions-and-classes.html#prod-ClassElement and parsing should fail.

**Comment by kaby76** [comment]
Yes, I just saw this last week while [scraping the Spec](https://github.com/kaby76/ecmascript-scrape). But, it turns out that Node, which is built on [V8](https://en.wikipedia.org/wiki/V8_(JavaScript_engine)), accepts the field with or without the semi-colon. So, what is "JavaScript"? How is [javascript ](https://github.com/antlr/grammars-v4/tree/master/javascript/javascript) and [ecmascript ](https://github.com/antlr/grammars-v4/tree/master/javascript/ecmascript) supposed to differ in this repo?

**Comment by ris58h** [comment]
Good question. I've checked Firefox and it allows a field declaration without semicolon so the code below is OK
```
class Foo {
    bar
    baz
}
```
but
```
class Foo {
    bar baz
}
```
is not OK and leads to
```
Uncaught SyntaxError: unexpected token: identifier debugger eval code:2:8
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Missing_semicolon_before_statement
